### DISCUSSION TITLE
Keep exif data when converting from jpg to webp

### DISCUSSION THREAD

**Discussion by SIARM8** [ImageMagick/ImageMagick#2683] [Help]
Hello, I would like to know if there's a way to keep the EXIF data (which is compatible with the webp format) when transcoding from jpg to webp format using ImageMagick.
I was able to do this by using the libwebp by Google and using "cwebp -metadata all" command. 
When examining the image using exiftool, the image includes vaild exif data. 
The same exif data is not shown if I check that same image file using the "identify -verbose" command of ImageMagick.
Thank you!

**Comment by snibgo** [comment]
When reading a file, IM does not record all exif data in memory, so not all exifdata will be written to the output file.

I suggest you use exifool "-tagsFromFile" for that purpose.
